### DISCUSSION TITLE
ORM return all fields

### DISCUSSION THREAD

**Discussion by mybigman** [adonisjs/core#3293] [Help]
How can I return all fields without specifying every one? Some of our vendor tables have 100+ columns :/

I can use `serializeExtras` but the problem with that it moves everything under `meta` and not the `root` which is problematic where some of the defined columns are at `root` level whilst the rest are under `meta`
```js 
public serializeExtras = true
```

Is there a clean way of handling this?

**Comment by feniljariwala82** [comment]
Can you please share your model code here, because when you query "Model.all()" it will fetch all the columns except "password". So I request you to share your code so that I can help you out.

**Comment by mybigman** [comment]
`Model.all()` returns all rows not all fields.

```js
import { BaseModel, column, HasOne, hasOne } from '@ioc:Adonis/Lucid/Orm'

export default class Order extends BaseModel {
  public static table = 'mytable'

  // public serializeExtras = true

  @column({ isPrimary: true })
  public orderId: number
}
```

**Comment by tsulatsitamim** [comment]
If you don't need relationship you can use Database Query Builder instead of Model.
you still can use join if needed

**Comment by legawa4201** [comment]
Im also facing this issue why we put the $extras column inside a meta object? I personally think this is a weird design decision

**Reply by RomainLanz** [reply]
> Im also facing this issue why we put the $extras column inside a meta object? I personally think this is a weird design decision

What's the issue you are having?
